packages/ui/tsconfig.json

```diff
@@ -8,7 +8,7 @@
       "@/*": ["./src/*"]
     },
     "module": "preserve",
-    "moduleResolution": "bundler"
+    "moduleResolution": "node"
```

Comment:
Make sure Amplify passes and we know what config we need to change for Vite.

Reply:
Reverting this change. This should be properly tested outside the scope of this PR